src/applications/vaos/appointment-list/components/cards/AppointmentStatus.jsx

```diff
@@ -52,6 +52,7 @@ export default function AppointmentStatus({
         <i
           aria-hidden="true"
           className={`fas ${iconClass} vads-u-line-height--4`}
+          style={{ color: '#2e8540' }}
```

Comment:
It looks like we don't apply a color to any of the other appointment icons.  I would check with Lenae and Ryan if they should all have color or not?

If so, you may want to use this color class instead of the hex code: `.vads-u-color--green`

Reply:
Removed color and horizontal line per Ryan.